server/routes/support/handlers/offenderDetail.ts

```diff
@@ -18,7 +23,43 @@ export default class OffenderDetailRoutes {
       ? await this.communityService.getStaffDetailByStaffCode(probationPractitioner?.staff.code)
       : undefined
     const hdcStatus = _.head(await this.prisonerService.getHdcStatuses([prisonerDetail], user))
+    const licenceSummary = await this.licenceServer.getLatestLicenceByNomisIdsAndStatus([nomsId], [], user)
 
+    let licenceFromSummary
+    let licencecrn = 'Not found'
+    let crd = 'Not found'
+    let actualReleaseDate = 'Not found'
+    let sentenceStartDate = 'Not found'
+    let sed = 'Not found'
+    let led = 'Not found'
+    let tussd = 'Not found'
+    let licenceTused = 'Not found'
```

Comment:
We're still relying on a lot of `let`s here

Reply:
Done.